### DISCUSSION TITLE
We are unable to complete your signup.

### DISCUSSION THREAD

**Discussion by hzn0kx** [vercel/community#4034] [Account Issue] [answered]
### Question

We are unable to complete your signup.

Real information, real account, registered but no way to log in

**Comment by hzn0kx** [comment]
my email hzn0kx@outmail.com

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
